chatter.js

```diff
@@ -1,7 +1,8 @@
 Chatter = {
   options: {
     messageLimit: 30,
-    nickProperty: "username",
+    roomLimit: 15,
```

Comment:
Is this the max number of rooms a user can have, or a limit to the number of users in a room, or messages in a room?